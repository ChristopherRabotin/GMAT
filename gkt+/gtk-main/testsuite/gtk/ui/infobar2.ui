<!-- invalid response value -->
<interface>
  <object class="GtkInfoBar">
    <child>
      <object class="GtkButton" id="button_cancel"/>
    </child>
    <action-widgets>
      <action-widget response="xyz">button_cancel</action-widget>
    </action-widgets>
  </object>
</interface>
