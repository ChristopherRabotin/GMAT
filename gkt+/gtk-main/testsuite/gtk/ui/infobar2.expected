ERROR: gtk-builder-error-quark 6
.:8:38 Could not parse enum: 'xyz'
